e2e-tests/test/evm-apis.test.ts

```diff
@@ -0,0 +1,102 @@
+import { expect } from "chai";
+import { getTestProvider } from "../helpers/utils";
+import { Wallet } from "zksync-web3";
+import { RichAccounts } from "../helpers/constants";
+import { ethers } from "ethers";
+
+const provider = getTestProvider();
+
+describe("evm_mine", function () {
+  it("Should mine one block", async function () {
+    // Arrange
+    const startingBlock = await provider.getBlock("latest");
+
+    // Act
+    await provider.send("evm_mine", []);
+
+    // Assert
+    const latestBlock = await provider.getBlock("latest");
+    expect(latestBlock.number).to.equal(startingBlock.number + 1);
```

Comment:
does it work after my PR? (as we might be mining 2 blocks..)

Reply:
Yes, I've already merged `main` into this branch